### PR TITLE
[FEATURE] preview page exists and displays read-only version of page 1

### PR DESCRIPTION
**Pull request by danWatkinson**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,5 +10,3 @@ git clone https://github.com/foundry4/fsa-incident-application-form.git
 cd fsa-incident-application-form
 npm test
 ```
-
```
Comment (danWatkinson) [inline README.md:13]: spotted a rogue "force a build" push from when i was showing Paul how the pipeline worked..

File: incident-application-form/views/preview.njk [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+{{ i18n.pageTitle[i18n.languageCode] }}
+{% endblock %}
+
+{% block content %}
+
+<form method="POST" action="" novalidate>
+
+    <h2>{{ i18n.sections.yourDetails[i18n.languageCode] }}</h2>
+    <dl class="govuk-summary-list govuk-summary-list--no-border">
+
```
Comment (danWatkinson) [inline incident-application-form/views/preview.njk:16]: these are all distressingly repetitive. if we have time we can just write a macro like the govUk ones to spit out the html.. from there it's only so big a leap to having a json file that describes our page and looping over it to work out what to display...
but 2 weeks so...

File: test/cypress/integration/user_views_preview.spec.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+
+      cy.get('[data-cy="address.country"]').select("Norway"); //?
+
+      cy.get('[data-cy="submit"]').click();
+
+      // shortcut; this is all meant to be on the session
+      //  so for individual pages we can prove their content in isolation
+      cy.visit(PREVIEW);
+
+
```
Comment (danWatkinson) [inline test/cypress/integration/user_views_preview.spec.js:49]: leaving passing tests  - but in a world where we are showing a summary it turns out we need to keep the 'display value' as well as the 'value' for these.. right now we show "06". I think it proves the point in terms of putting data in and getting it out, but like i say the dropdowns need to record the display-value of what was selected as well as the 'value' so we can display something useful in the summary.

File: incident-application-form/views/preview.njk (hunk 2 above)
Comment (a6software) [reply incident-application-form/views/preview.njk:16]: yeah there's a ton of repetition on my part, not ideal but also not had time to think properly about better abstractions at this stage.

**Review by a6software — APPROVED** [review]

**Pull request merged**